apps/functions-subscription/local.settings.json.example

```diff
@@ -17,22 +17,26 @@
 
     "SUBSCRIPTION_HISTORY_CONSUMER": "on",
     "SUBSCRIPTION_HISTORY_COSMOSDB_CONTAINER_NAME": "subscription-history",
+    # On local development, prefer the connection string instead of the account endpoint
+    "SubscriptionHistoryCosmosConnection": "<put-here-the-the-cosmos-database-connection-string>",
     "SubscriptionHistoryCosmosConnection__accountEndpoint": "<put-here-the-the-cosmos-database-account-endpoint>",
 
     "SUBSCRIPTION_REQUEST_CONSUMER": "on",
     "SUBSCRIPTION_REQUEST_EVENTHUB_NAME": "<put-here-the-name-of-the-event-hub>",
+    "SubscriptionRequestEventHubConnection__fullyQualifiedNamespace": "<put-here-the-the-event-hub-fully-qualified-namespace>",
 
     "EVENTS_PRODUCER": "on",
     "EVENTS_SERVICEBUS_TOPIC_NAME": "<put-here-the-the-service-bus-topic-name>",
-    "EventProducerCosmosConnection__accountEndpoint": "<put-here-the-the-cosmos-database-account-endpoint>",
```

Comment:
**note**: this is not used nor referenced anywhere in the code.

On the App Function resources, this variable is not present.